### ISSUE TITLE
Sourcemaps with autoprefixer on windows

### ISSUE THREAD

**Issue by JosephSilber** [dlmanning/gulp-sass#296]
When trying to get sourcemaps for sass + autoprefixer on Windows, the resulting sourcemap's `contentSource` is the compiled CSS, not the original sass.

```js
var autoprefixer = require('gulp-autoprefixer');
var gulp = require('gulp');
var rename = require('gulp-rename');
var sass = require('gulp-sass');
var sourcemaps = require('gulp-sourcemaps');

gulp.task('css', function () {
    gulp.src('resources/assets/sass/app.scss')
        .pipe(sourcemaps.init())
        .pipe(sass({ outputStyle: 'compressed' }))
        .pipe(autoprefixer())
        .pipe(rename('css/app.css'))
        .pipe(sourcemaps.write('.'))
        .pipe(gulp.dest('public'));
});
```

Running `autoprefixer` without `sass` results in proper sourcemaps.

---

For more context, see here: http://stackoverflow.com/questions/30805146

---

To test this on a Windows machine, download this: [gulp-sass-autoprefixer-sourcemaps.zip](gulp-sass-autoprefixer-sourcemaps.zip).

Unzip, and run: `npm install && gulp css`.

**Comment by lagun4ik** [comment]
```
var myth = require('gulp-myth');
var gulp = require('gulp');
var rename = require('gulp-rename');
var sass = require('gulp-sass');
var csso = require('gulp-csso');
var sourcemaps = require('gulp-sourcemaps');

gulp.task('css', function () {
    gulp.src('resources/assets/sass/app.scss')
        .pipe(sourcemaps.init())
        .pipe(sass({errLogToConsole: true}))
        .pipe(myth())
        .pipe(csso())
        .pipe(rename('css/app.css'))
        .pipe(sourcemaps.write('.'))
        .pipe(gulp.dest('public'));
});
```

**Comment by Snugug** [comment]
What version of Gulp Sass are you using? Does it happen if you reduce your pipe to just Sass and Sourcemaps?

**Comment by MaciejJanyska** [comment]
I'm using the newest version of gulp-sass and have the same issue on Windows 7. When autoprefixer is used, "sources" is my compiled css. Without it, everything is fine and sources include sass files.

**style.css.map (with autoprefixer):**

```
{"version":3,"sources":["style.css"],"names":[] (...)
```

**style.css.map (without autoprefixer):**

```
{"version":3,"sources":["_settings.scss","libs/bootstrap/_normalize.scss","libs/bootstrap/_print.scss", (...)
```

**Comment by TrySound** [comment]
@all #314

**Comment by Snugug** [comment]
Resolved by #314
